tanner/emulators/xxe_injection.py

```diff
@@ -29,12 +34,25 @@ def __init__(self, loop=None):
         return xxe_injection_result
 
     def scan(self, value):
+        """
+        Scans the input payload to detect attack using regex
+        :param value (str): code from attacker
+        :return: detection (dict): name (attack name), order (attack order) as keys
+        """
+
         detection = None
         if patterns.XXE_INJECTION.match(value):
             detection = dict(name='xxe_injection', order=3)
         return detection
 
     async def handle(self, attack_params):
+        """
+        Handler of emulator
+        :param attack_params (list): contains dicts as elements with id and value (payload from attacker) as keys
+        :return: dict: value (result of emulator), page (if set to true the payload will be injected to index.html
```

Comment:
`(dict)`
for consistency with your style of naming types :)

Reply:
Thanks 😅